File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Print row for left pyramid
    for (int i = 0; i < height; i++)
    {
        // Print spaces for left pyramid
        for (int j = i + 1; j < height; j++)
        {
            printf(" ");
        }

        // Print bricks for left pyramid
        for (int n = 0; n <= i; n++)
        {
            printf("#");
        }

        // Print 2 spaces between the two pyramids
        printf("  ");

        // Print bricks for right pyramid
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: Week 6 - Python/Problem Set 6/sentimental-mario-more/mario.py
from cs50 import get_int


# TODO
height = get_int('Height: ')
while height < 1 or height > 8:
    print('Only height between 1 and 8, inclusive, allowed.')
    height = get_int('Height: ')

for row in range(height):
    bricks = '#' * (row + 1)
    print(' ' * (height - row - 1), end='')
    print(bricks, end='')
    # gap between two pyramid halves
    print('  ', end='')
    print(bricks)